editor.planx.uk/src/@planx/components/List/Public/tests/index.listBuilding.test.tsx

```diff
@@ -11,6 +12,21 @@ import { fillInResponse } from "./testUtils";
 
 Element.prototype.scrollIntoView = vi.fn();
 
+const mocks = vi.hoisted(() => {
```

Comment:
I spent a while trying to move these repeated mocks of file uploads to either a helper function or just into `testUtils.ts` directly but couldn't quite get it to work. I think the issue is the order in which imports and mocks are hoisted.

This means there's a bit of repetition each place we need to use `fillInResponse()` - if anybody knows ways to avoid this and directly define a mock which will be correctly hoisted please let me know.